charts/partials/templates/_proxy-init.tpl

```diff
@@ -0,0 +1,40 @@
+{{- define "partials.proxy-init" -}}
+- args:
+  - --incoming-proxy-port
+  - {{.Proxy.Ports.Inbound | quote}}
+  - --outgoing-proxy-port
+  - {{.Proxy.Ports.Outbound | quote}}
+  - --proxy-uid
+  - {{.Proxy.UID | quote}}
+  - --inbound-ports-to-ignore
+  - {{.Proxy.Ports.Control}},{{.Proxy.Ports.Admin}}{{ternary (printf ",%s" .ProxyInit.IgnoreInboundPorts) "" (not (empty .ProxyInit.IgnoreInboundPorts))}}
+  - --outbound-ports-to-ignore
+  - {{.ProxyInit.IgnoreOutboundPorts | quote}}
```

Comment:
From looking a the test fixtures, we're not rendering this option at all if there are no outbound ports to ignore. So I suggest we change this to:
```
{{- with .ProxyInit.IgnoreOutboundPorts }}
- --outbound-ports-to-ignore
- {{. | quote}}
{{- end }}
```

Reply:
Ah.. right. On the control plane side, there will always be `443`, but not the rest of the data plane. I will likely use an `if` instead of `with`, because with `with`, if it's missing during `helm install`, it will fail
```
[ERROR] templates/: render error in "linkerd/templates/config.yaml": template: linkerd/templates/config.yaml:28:6: executing "linkerd/templates/config.yaml" at <include "linkerd.configs.proxy" .>: error calling include: template: linkerd/templates/_config.tpl:32:51: executing "linkerd.configs.proxy" at <.ProxyInit.IgnoreOutboundPorts>: wrong type for value; expected string; got interface {}
```